packages/serialize/src/index.ts

```diff
@@ -6,6 +6,8 @@ import * as CSS from 'csstype'
 
 export type { RegisteredCache, SerializedStyles }
 
+declare const process: { env: { NODE_ENV: string } }
```

Comment:
When this was creating an issue (I mean - during what task etc). I believe that I was fixing similar (but maybe different) problem by adding this to the files seen by dtslint:
```ts
/// <reference types="node" />
```

Like here:
https://github.com/emotion-js/emotion/blob/9ca22c6c23e9effa086d161a9b0ae1c645686680/packages/serialize/types/index.d.ts#L1

Reply:
Are you sure referencing all of the node types is the best solution? Because in the browser, we don't have access to Node stuff, except `process.env.NODE_ENV` which I believe is set by the bundler e.g. Webpack.